challenges/intermediate/crack-caesar-cipher/solution/fdb86/caesar-cipher.ts

```diff
@@ -0,0 +1,21 @@
+export function encode(word: string, shift: number) {
+
+  shift = shift % 26;
+  
+  return word.split("").map(caesar => {
+    const ascii: number = caesar.charCodeAt(0);
+    const new_code: number = ascii + shift;
+    if (ascii >= 65 && ascii <= 90) {
```

Comment:
What is `65`? What is `90`? Consider adding two constant explaining this value meaning with their names

Reply:
new const created